frontend/src/Onboarding/TransferCoursesScreen.tsx

```diff
@@ -0,0 +1,363 @@
+import React, { Component } from "react";
+import { connect } from "react-redux";
+import { AppState } from "../state/reducers/state";
+import { Dispatch } from "redux";
+import {
+  ScheduleCourse,
+  Major,
+  IRequiredCourse,
+  Requirement,
+} from "../../../common/types";
+import {
+  getDeclaredMajorFromState,
+  getCompletedRequirementsFromState,
+} from "../state";
+import {
+  setCompletedCourses,
+  setTransferCourses,
+} from "../state/actions/scheduleActions";
+import styled from "styled-components";
+import { fetchCourse } from "../api";
+import { NextButton } from "../components/common/NextButton";
+import { Link, withRouter, RouteComponentProps } from "react-router-dom";
+import { GenericOnboardingTemplate } from "./GenericOnboarding";
+import CheckBoxIcon from "@material-ui/icons/CheckBox";
+import CheckBoxOutlineBlankIcon from "@material-ui/icons/CheckBoxOutlineBlank";
+import {
+  Link as ButtonLink,
+  Collapse,
+  Grid,
+  Paper,
+  Checkbox,
+} from "@material-ui/core";
+import { AddClassModal } from "../components/AddClassModal";
+import { AddBlock } from "../components/ClassBlocks/AddBlock";
+
+const MainTitleText = styled.div`
+  font-size: 18px;
+  margin-left: 4px;
+  margin-top: 10px;
+  margin-bottom: 12px;
+  font-weight: 500;
+`;
```

Comment:
I noticed these styled divs for the most part closely resemble those also on the `CompletedCoursesScreen`, do we want to move these into a common file for reuse? Maybe `GenericOnboarding.tsx`?

Reply:
Ooh fact, but we'll probably make that a separate tech debt ticket since this story is already kinda thicc